### PR TITLE
Add new GC members, remove bootstrap GC

### PR DESCRIPTION
**Pull request by DavidPHirsch**
adding gc to config, and org approvers

### CODE REVIEW

**Comment by DavidPHirsch** [comment]
@aepfli any suggestions on what I did wrong?

File: config/open-feature/org.yaml [hunk 1]
```diff
@@ -34,6 +35,7 @@ members:
   - agentgonzo
   - ajhelsby
   - alina-v1
+  - AlexJones
```
Comment (aepfli) [inline config/open-feature/org.yaml:38]:
```suggestion
  - AlexsJones
```

you do have a typo in this user name

**Comment by aepfli** [comment]
> @aepfli any suggestions on what I did wrong?
https://github.com/open-feature/community/pull/342#discussion_r1478323361 

there is a typo in the username, thats why it is complaining :)

File: config/open-feature/org.yaml (hunk 1 above)
Comment (DavidPHirsch) [reply config/open-feature/org.yaml:38]: Thank you, thought I had messed other stuff up

**Review by aepfli — APPROVED** [review]

**Review by dabeeeenster — APPROVED** [review]

**Review by beeme1mr — APPROVED** [review]

File: areas-of-interest.md
Comment (beeme1mr) [inline areas-of-interest.md:1]: This was unused and stale anyways. I removed the file and the reference in the readme.

File: Elections/2023/Candidates.md [hunk 2]
```diff
@@ -57,6 +57,7 @@ In alphabetical order:
 - Company: Google
 - GitHub: [oxddr](https://github.com/oxddr)
 - Description: Working on Google-internal platform for feature flagging and A/B testing. Interested in OpenFeature from the perspective of the end-used that happen to be a large enterprise. I have an experience working with large-scale backend services and building a feature flag solution for them. I believe feature flags are simple and effective mean to improve software reliability and thus I strongly believe in OpenFeature initiative.
+
```
Comment (beeme1mr) [inline Elections/2023/Candidates.md:60]: This was causing a linting issue in the docs.

File: governance-charter.md
Comment (beeme1mr) [inline governance-charter.md:1]: Removed bootstrap language from the GC docs.

**Review by beeme1mr — APPROVED** [review]

**Review by jonathannorris — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -5,15 +5,16 @@
 Table of Contents
 </summary>
 
-- [Getting Involved](#getting-involved)
+- [Getting involved](#getting-involved)
```
Comment (toddbaert) [inline README.md:8]:
```suggestion
- [Getting Involved](#getting-involved)
```

All the others use full title case.

File: governance-charter.md [hunk 4]
```diff
@@ -2,25 +2,18 @@
...
 ## Goals
 
-The initial role of the governance committee is to **instantiate the formal
-process for OpenFeature governance**. In addition to defining the initial
-governance process, the bootstrap committee strongly believes that **it is
-important to provide a means for iterating** the processes defined by the
-governance committee. We do not believe that we will get it right the first
-time, or possibly ever, and won’t even complete the governance development in a
-single shot. The role of the governance committee is to be a live, responsive
+The role of the governance committee is to be a live, responsive
```
Comment (toddbaert) [inline governance-charter.md:10]: Capitalization of "governance committee" is not consistent.

File: governance-charter.md [hunk 5]
```diff
@@ -30,18 +23,11 @@ and organizing elections for the elected governance board body.
...
-
-
 ## Decision making
 
 Decisions are made by a consensus of _Project Members_ and [Interested Parties](./interested-parties.md).
 If this consensus cannot be reached,
-the decision can be made by the plain majority vote of _Bootstrap Governance Committee Members_.
-
-<!-- TODO: List founding members or delegate the decision to CDF TAG App Delivery or another entity -->
+the decision can be made by the plain majority vote of _Governance Committee Members_.
```
Comment (toddbaert) [inline governance-charter.md:30]:
```suggestion
the decision can be made by the plain majority vote of _Governance Committee_ members.
```

File: governance-charter.md [hunk 6]
```diff
@@ -119,8 +68,7 @@ faith. If this turns out to be a problem, this will be reconsidered.
 
 ### Eligibility for voting
 
-All Members are eligible to vote for the governance committee
-members. 
+All Members are eligible to vote for the governance committee members.
```
Comment (toddbaert) [inline governance-charter.md:71]:
```suggestion
All organization members are eligible to vote for the governance committee members. 
```

**Review by toddbaert — APPROVED** [review]
Minor editorial comments.

**Pull request merged**